.github/workflows/test.yml

```diff
@@ -35,17 +35,20 @@ jobs:
           # Minimum supported rust version (MSRV)
           - "georust/geo-ci:rust-1.63"
           # Two most recent releases - we omit older ones for expedient CI
-          - "georust/geo-ci:proj-9.2.1-rust-1.71"
-          - "georust/geo-ci:proj-9.2.1-rust-1.72"
+          - "ghcr.io/georust/geo-ci:proj-9.6.0-rust-1.83"
+          - "ghcr.io/georust/geo-ci:proj-9.6.0-rust-1.84"
     container:
       image: ${{ matrix.container_image }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
-      - run: cargo install --version 1.6.0 cargo-all-features
-      - run: cargo build-all-features
-      - run: cargo test-all-features
-      - run: cargo build -p rstar-benches
+      - name: Setup Rust cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: "rstar -> target"
+          key: ${{ matrix.container_image }}
+      - run: cargo check --all-targets
+      - run: cargo test
```

Comment:
Shouldn't these have a `--workspace` added?

Also, the `cargo build -p rstar-benches` command does not seem to covered, is this intentional?